core/ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -402,7 +434,17 @@
       ],
       "name": "default",
       "defaultEnv": "default",
-      "description": null
+      "description": null,
+      "commands": [
+        {
+          "name": "run",
```

Comment:
I'm not sure that this command is useful for every projects as name of app is different and we may use other tools like grunt gulp etc